core/templates/dev/head/pages/topic-editor-page/navbar/topic-editor-navbar.directive.ts

```diff
@@ -19,9 +19,8 @@
 require(
   'components/common-layout-directives/common-elements/' +
   'loading-dots.directive.ts');
-
```

Comment:
Add the empty line back. We keep empty line between  `requires` needed for the related HTML fiel and the TS file.

Reply:
Done.